migrations/migration_0001_move_selected_collections.py

```diff
@@ -0,0 +1,40 @@
+"""
+Migrate from selected_collections inside the schema, to a global one in its own summaries schema.
+https://github.com/open-contracting/kingfisher-summarize/issues/197
+
+Run this with `python -m migrations.migration_0001_move_selected_collections` in the parent directory.
+
+"""
+
+from psycopg2.errors import UndefinedTable
+
+from ocdskingfishersummarize.db import Database
+
+
+def migrate():
+    db = Database()
+
+    db.execute('CREATE SCHEMA IF NOT EXISTS summaries')
+    db.set_search_path(["summaries"])
+    db.execute("""CREATE TABLE IF NOT EXISTS selected_collections
+                  (schema TEXT NOT NULL, collection_id INTEGER NOT NULL)""")
+    db.execute("""CREATE UNIQUE INDEX IF NOT EXISTS selected_collections_schema_collection_id
+                  ON selected_collections (schema, collection_id)""")
+    db.commit()
+
+    for schema in db.schemas():
+        print(schema)
+        db.set_search_path([schema])
+        try:
+            collections = db.pluck('SELECT id FROM selected_collections')
+        except UndefinedTable as e:
+            print(e)
+            db.commit()
+            continue
+        db.execute_values('INSERT INTO summaries.selected_collections (schema, collection_id) VALUES %s',
+                          [(schema, _id,) for _id in collections])
```

Comment:
We could delete the existing table here, but I haven't so it doesn't break old notebooks etc. If we think it's more important to have all the schemas be consistent, then I'm happy to change this.